index.js

```diff
@@ -17,6 +19,9 @@ module.exports = {
             remove_filter: 'Bỏ bộ lọc',
             back: 'Trở về',
             bulk_actions: '%{smart_count} đã chọn',
+            clear_input_value: 'Xóa giá trị',
+            clone: 'nhân bản',
```

Comment:
I saw the english version uppercase for the first letter. I think vietnamese version also need uppercase for first letter.